src/components/date-picker/flatpickr-adapter/flatpickr-adapter.tsx

```diff
@@ -175,6 +176,11 @@ export class DatePickerCalendar {
         this.flatPickrCreated = true;
     }
 
+    @Method()
+    public open() {
+        this.picker.open();
```

Comment:
This commit ("do not open flatpickr on click, but use timeout after portal has opened") is likely not needed. I just wanted to push it before removing it just in case it turned out it was necessary after all. After some quick testing, it seems not to be.